static/src/js/components/AdminPreferencesMetrics/AdminPreferencesMetricsList.jsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable react/prop-types */
```

Comment:
Don't add eslint-disable lines for a full file. You should never add this specific ignore, just define the prop types. If you absolutely have to ignore eslint it should only be for the specific line, and never the full file.